tests/new_unit_tests/Commands/Connection/InfoCommandTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Pantheon\Terminus\Commands\Connection\InfoCommand;
+use Pantheon\Terminus\Config;
+
+use Prophecy\Prophet;
+use Terminus\Models\Environment;
+use Terminus\Models\Site;
+use VCR\VCR;
+
+/**
+ * Test suite for class for Pantheon\Terminus\Commands\Connection\InfoCommand
+ */
+class InfoCommandTest extends ConnectionCommandTest
+{
+    private $prophet;
+
+    /**
+     * Test suite setup
+     *
+     * @return void
+     */
+    protected function setup()
+    {
+        parent::setUp();
+
+        // TODO: (ajbarry) Where is the appropriate place to configure VCR mode?
+        // Picking up 'new_episodes' somewhere
+        VCR::configure()->setMode('none');
+
+        $this->command = new InfoCommand(new Config());
```

Comment:
If it pleases you, you can use $this->getConfig() to retrieve an inherited config object.